code/modules/clothing/spacesuits/rig/rig.dm

```diff
@@ -363,7 +363,7 @@
 				M.drop_from_inventory(piece)
 			piece.forceMove(src)
 
-	if(active == TRUE) // dains power from the cell whenever the suit is sealed
+	if(active && cell) // dains power from the cell whenever the suit is sealed
```

Comment:
pay no attention to this change.
https://github.com/discordia-space/CEV-Eris/pull/5219/files